prime-router/settings/staging/0149-etor.yml

```diff
@@ -49,6 +49,8 @@
       jurisdictionalFilter:
         - "%hl7MessageType = 'O21'"
       #  - "%performerState = 'CO' or %patientState = 'CO'"
+      qualityFilter:
+        - "%hl7MessageType = 'O21'"
```

Comment:
It might be worth adding a check for `hl7MessageType.exists()`. We've been doing that for some of the conditions.